person_class.rb

```diff
@@ -4,14 +4,15 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = id
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   private
```

Comment:
- [ ] As required for the project the a person to have many rentals, kindly ensure that: `Getters` and `Setters` for `@rentals` are present, and you should also have a method for adding rentals where the person itself should is sent as a parameter to create the new rental achieving the 'has-many' association. It should be similar to this:
```ruby
class Person
  attr_accessor :name, :age, :rentals
  .
  .

  def initialize(age, name = "Unknown", parent_permission = true)
    .
    .
    @rentals = []
  end

  def add_rental(book, date)
    Rental.new(date, book, self)
  end

  .
  .
end
```